### PR TITLE
[16.0][mig] analytic_partner

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**

### CODE REVIEW

**Review by flotho — APPROVED** [review]
code review
LGTM

**Comment by rafaelbn** [comment]
@heliaktiv could you please rebase?

**Review by dreispt — APPROVED** [review]

File: analytic_partner/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Link analytic items and partner",
     "summary": "Search and group analytic entries by partner",
-    "version": "14.0.1.0.0",
+    "version": "16.0.0.1.0",
```
Comment (rousseldenis) [inline analytic_partner/__manifest__.py:9]: `16.0.1.0.0` is the default value.

**Comment by rousseldenis** [comment]
@heliaktiv

File: analytic_partner/__manifest__.py (hunk 1 above)
Comment (maisim) [reply analytic_partner/__manifest__.py:9]: Hi! Can someone make this little modification online?

File: .pre-commit-config.yaml
Comment (maisim) [inline .pre-commit-config.yaml:1]: I guess this file should not be included in the pr?

**Comment by maisim** [comment]
Ok just re-discovered this one was replaced by the #609 :roll_eyes:
